### PR TITLE
SonarCloud issue fixes

### PR DESCRIPTION
**Pull request by mcpierce**
## Status
**READY**

## Migrations
NO

## Description
Fixed a few issues reported by SonarCloud.

### CODE REVIEW

File: comixed-library/src/main/java/org/comixed/adaptors/archive/SevenZipArchiveAdaptor.java [hunk 1]
```diff
@@ -69,8 +69,11 @@ private void addFileToArchive(SevenZOutputFile archive, String filename, byte[]
     archive.write(content);
     archive.closeArchiveEntry();
 
-    this.log.debug("Cleaning up the temporary file");
-    tempFile.delete();
+    if (tempFile.delete()) {
+      this.log.debug("Cleaning up the temporary file");
```
Comment (BRUCELLA2) [inline comixed-library/src/main/java/org/comixed/adaptors/archive/SevenZipArchiveAdaptor.java]: lombok generate this logger : 
private static final Logger log = org.apache.logging.log4j.LogManager.getLogger(SevenZipArchiveAdaptor.class);
so I think we can access to the log without the "this".

File: comixed-library/src/main/java/org/comixed/adaptors/archive/SevenZipArchiveAdaptor.java [hunk 2]
```diff
@@ -69,8 +69,11 @@ private void addFileToArchive(SevenZOutputFile archive, String filename, byte[]
...
+    } else {
+      this.log.debug("Failed to delete temporary file: {}", tempFile.getAbsolutePath());
```
Comment (BRUCELLA2) [inline comixed-library/src/main/java/org/comixed/adaptors/archive/SevenZipArchiveAdaptor.java]: lombok generate this logger : 
private static final Logger log = org.apache.logging.log4j.LogManager.getLogger(SevenZipArchiveAdaptor.class);
so I think we can access to the log without the "this".

**Review by BRUCELLA2 — CHANGES_REQUESTED** [review]
Quality Gate failed, we are < 80% of coverage of New Code. However, in view of what is not covered I think 76% coverage is acceptable. Do you agree with that?

File: comixed-library/src/main/java/org/comixed/adaptors/archive/SevenZipArchiveAdaptor.java (hunk 1 above)
Comment (mcpierce) [reply comixed-library/src/main/java/org/comixed/adaptors/archive/SevenZipArchiveAdaptor.java]: Could you do a global search-and-replace to replace "this.log" in the Java code with "log" and submit a PR?

Comment (BRUCELLA2) [reply comixed-library/src/main/java/org/comixed/adaptors/archive/SevenZipArchiveAdaptor.java]: Yes i can.

**Comment by mcpierce** [comment]
@BRUCELLA2 I've added some additional changes to hopefully wrap up the SonarCloud issues.

**Review by BRUCELLA2 — APPROVED** [review]
It's ok for me. I approve this PR

**Review by BRUCELLA2 — APPROVED** [review]

**Review by BRUCELLA2 — APPROVED** [review]

**Pull request merged**